pages/api/token/temporary.ts

```diff
@@ -143,4 +160,114 @@ const sendTemporaryTokenByEmail = async (email: string, temporaryToken: string)
     });
 };
 
-export default middleware([cors({ allowedMethods: ["POST"] }), validBearerToken()], handler);
+const requestContainsEmail = (): MiddlewareRequest => {
```

Comment:
Did this really needed to be seperated out into it's own middleware function?
It was previously in the scope of the existing function with:
```js
    const email = req.body["email"];
    if (!email) {
      return res.status(400).json({ error: "Invalid payload" });
    }
```

Reply:
The reason I did that is because the next middleware function I created (`canLogIn`) needs an email address to run its check. The `handler` function is the last piece of code being executed in the temporary token API logic.